app/validators/email_validator.rb

```diff
@@ -0,0 +1,9 @@
+# app/validators/email_validator.rb
+
+class EmailValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    unless value =~ /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i

```

Comment:
True. Seems like the only way is with the confirmation. The only think  you can get with the validation is something that looks like an email. I´ll review it when we start working with the mailers.